integration-tests/api/__tests__/admin/return-reason.js

```diff
@@ -151,15 +406,170 @@ describe("/admin/return-reasons", () => {
           },
         })
         .catch((err) => {
-          console.log(err);
-        });
+          console.log(err)
+        })
 
-      expect(response.status).toEqual(200);
+      expect(response.status).toEqual(200)
       expect(response.data.return_reasons).toEqual([
         expect.objectContaining({
           value: "too_big",
         }),
-      ]);
-    });
-  });
-});
+      ])
+    })
+  })
+
+  describe("DELETE /admin/return-reasons", () => {
+    beforeEach(async () => {
+      try {
+        await adminSeeder(dbConnection)
+      } catch (err) {
+        console.log(err)
+        throw err
+      }
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it('deletes single return reason', async () => {
+      expect.assertions(6)
+
+      const api = useApi()
+
+      const payload = {
+        label: "Too Big",
+        description: "Use this if the size was too big",
+        value: "too_big",
+      }
+
+      const response = await api
+        .post("/admin/return-reasons", payload, {
+          headers: {
+            Authorization: "Bearer test_token",
+          },
+        })
+        .catch((err) => {
+          console.log(err)
+        })
+
+      expect(response.status).toEqual(200)
+
+      expect(response.data.return_reason).toEqual(
+        expect.objectContaining({
+          label: "Too Big",
+          description: "Use this if the size was too big",
+          value: "too_big",
+        })
+      )
+        
+      const deleteResult = await api.delete(`/admin/return-reasons/${response.data.return_reason.id}`, {
+        headers: {
+          Authorization: "Bearer test_token",
+        },
+      })
+
+      expect(deleteResult.status).toEqual(200)
+
+      expect(deleteResult.data).toEqual({id: response.data.return_reason.id, object: 'return_reason', deleted: true})
```

Comment:
too wide - please insure that you have a prettier plugin installed in your editor.